lib/index.js

```diff
@@ -288,24 +294,18 @@ Reactive.prototype._bind = function() {
 Reactive.prototype.bind = function(name, fn) {
   var self = this;
   if ('object' == typeof name) {
-    for (var key in name) {
-      this.bind(key, name[key]);
-    }
-    return;
+    Object.keys(name).forEach(function (key) {
```

Comment:
IE9+ should be fine. There are shims for IE8: see the reactive-ie8-shims.
We haven't changed the name yet
On Mar 16, 2015 9:26 PM, "Stephen Mathieson" <notifications@github.com>
wrote:

> In lib/index.js
> <https://github.com/notacatjs/dommit/pull/3#discussion_r26547051>:
>
> > @@ -288,24 +294,18 @@ Reactive.prototype._bind = function() {
> >  Reactive.prototype.bind = function(name, fn) {
> >    var self = this;
> >    if ('object' == typeof name) {
> > -    for (var key in name) {
> > -      this.bind(key, name[key]);
> > -    }
> > -    return;
> > +    Object.keys(name).forEach(function (key) {
>
> IE will explode here
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/notacatjs/dommit/pull/3/files#r26547051>.
>